src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -450,28 +454,27 @@ public void onChange(final List<ResourceChange> changes) {
             final ResourceChange.ChangeType type = rc.getType();
             final String path = rc.getPath();
 
-            log.debug("onChange, type={}, path={}", rc.getType(), path);
+            log.debug("onChange, type={}, path={}", type, path);
 
             // don't care for system area
             if (path.startsWith(JCR_SYSTEM_PREFIX)) {
                 continue;
             }
 
-            // during startup: just enqueue the events
-            if (inStartup) {
-                if (type == ResourceChange.ChangeType.REMOVED
-                        || type == ResourceChange.ChangeType.ADDED
-                        || type == ResourceChange.ChangeType.CHANGED) {
+            boolean queued = false;
+
+            if (RELEVANT_CHANGE_TYPES.contains(type)) {
+                // during startup: just enqueue the events
+                if (inStartup) {
```

Comment:
can you merge these 2 conditions (and probably change the order, as the boolean check is faster)?